### ISSUE TITLE
Clicking link in the app should open browser window with the clicked link or a action should take place

### ISSUE THREAD

**Issue by ms10398** [fossasia/susi_desktop#106]
**Actual Behaviour**

When we click on any link in the electron app nothing happens

**Expected Behaviour**

When you click on a link in the app a browser window should open with the clicked link.

**Steps to reproduce it**

- Open the app
- Type `What is FOSSASIA` or any other query which generates a link
- Click on the link

##### Please check the applicable fields -

- [x] I ensure that this issue is not a duplicate of any current open issues(s).
- [x] Assign this issue to me (I want to work on this).
- [x] I will send my PR only after I'm assigned to this issue by any maintainer.
- [x] I understand that I will be unassigned from this issue if I fail to send a PR for this within a week.

**Comment by anshumanv** [comment]
Great Pick! 
https://github.com/electron/electron/blob/master/docs/api/shell.md#shellopenexternalurl-options-callback

**Comment by ms10398** [comment]
yes I searched and got to know there are many ways of doing it and one of the ways is that the way you highlighted.

I will make a single PR for the two issues as they are relatively small issues in the project.

**Comment by anshumanv** [comment]
Woah! I didn't know there are many ways, what else can we use?

**Comment by ms10398** [comment]
We can use node-open package 

but I think the way you specified is the best way to go as it is inbuilt in electron and dependency free :smile: 

and I had already tested your way and it worked perfectly.

**Comment by anshumanv** [comment]
Woah awesome! :+1:
